Frontend/student-hub/src/pages/Dashboard.js

```diff
@@ -33,6 +33,11 @@ function Dashboard() {
     const [searchTerm, setSearchTerm] = useState('');
     const [filteredData, setFilteredData] = useState(null);
     const [openCreateModal, setOpenCreateModal] = useState(false);
+    const [openCommentsModal, setOpenCommentsModal] = useState(false);
+    const [currentComments, setCurrentComments] = useState([]);
+    const [currentPostId, setCurrentPostId] = useState(null);
+    const [newComment, setNewComment] = useState('');
+
```

Comment:
since this isn't relevant to Verification this could be in a separate PR/Commit

Reply:
Ok I will do that.